src/include/planner/logical_plan/logical_operator/side_way_info_passing.h

```diff
@@ -8,6 +8,9 @@ namespace planner {
 enum class SidewaysInfoPassing : uint8_t {
     NONE = 0,
     LEFT_TO_RIGHT = 1,
```

Comment:
I wonder if we should always directly say PROBE_TO_BUILD, BUILD_TO_PROBE, instead of LEFT_TO_RIGHT, and RIGHT_TO_LEFT? The former one seems more clear, when there are cases we kind of mix the terms of build, probe and left, right altogether.